sources/packages/web/src/components/common/IdleTimeChecker.vue

```diff
@@ -32,22 +46,35 @@ export default {
     },
   },
   setup(props: any) {
-    const lastActivityLogin = ref(new Date());
     const interval = ref();
     const extendTimeModal = ref({} as ModalDialog<boolean>);
     const { isAuthenticated } = useInstitutionAuth();
     const { getDatesDiff } = useFormatters();
+    const { executeLogout } = useAuth();
+    const countdown = ref(COUNT_DOWN_TIMER_FOR_LOGOUT);
+    const countdownInterval = ref(0);
+    const modalOpen = ref(false);
+    const idleTimeInSeconds = ref(0);
 
-    const minimumIdleTime = computed(() => {
+    onMounted(async () => {
+      setLastActivityTime();
+      resetIdleCheckerTimer();
+      localStorage.removeItem(LOGGED_OUT_LOCAL_STORAGE_ITEM);
```

Comment:
is the purpose of this localStorage `LOGGED_OUT_LOCAL_STORAGE_ITEM` to communicate with the other tabs?

Reply:
yes, all the browser instances check if the variable is true and logout.